cmdline-pull/test_pubsub_sample.py

```diff
@@ -85,3 +85,46 @@ def test_list_subscriptions(self):
             main(['pubsub_sample.py', get_project_id(), 'list_subscriptions'])
         output = out.getvalue().strip()
         self.assertTrue(expected_sub in output)
+
+
+class PubSubPublishAndPullTestCase(unittest.TestCase):
```

Comment:
Can you merge these two classes? The motivation is to reduce the admin operations(create/delete).